docs/api/en/materials/LineDashedMaterial.html

```diff
@@ -12,7 +12,8 @@
 		<h1>[name]</h1>
 
 		<p class="desc">
-			A material for drawing wireframe-style geometries with dashed lines.
+			A material for drawing wireframe-style geometries with dashed lines.<br />
+			Note: Make sure to call [page:Line.computeLineDistances]() when using [name].
```

Comment:
An argument can be made that "Be sure to call" is more appropriate here. But I think a better way to say it is:

> Note: You must call [page:Line.computeLineDistances]() when using [name].